Data Plus and related changes (#7664)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

Our New Relic One pricing model is meant to be simple and to provide a lot of value even when you're [using New Relic for free](#free). Our pricing model bills on these factors:

> * Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. 100 GBs per month is free. The price per GB above that can vary depending on which [data plan](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you're on.

Reviewer: ```suggestion
* Your [ingested data](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing). We bill on data ingested, no matter the type, making your billing easy to understand. You'll receive 100 GBs per month for free. The price per GB above the free 100 GBs can vary depending on which [data plan](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices) you're on.
```

Clarity edit.
